packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Actions.js

```diff
@@ -53,18 +54,18 @@ const Actions = ({
         test-id={`${blockClass}__customize-columns-modal--reset`}
       >
         {resetToDefaultLabel}
-      </Button>
+      </Button> */}
```

Comment:
commented out code should really be cleaned out (there are of course exceptions).